core/src/toga/app.py

```diff
@@ -1020,6 +1020,16 @@ def set_full_screen(self, *windows: Window) -> None:
     # End backwards compatibility
     ######################################################################
 
+    def focus(self):
```

Comment:
The methods on this class are sorted; this should be added to the "Window control" section, alphabetically sorted in that section.